src/WsExportException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App;
+
+use Exception;
+
+class WsExportException extends Exception {
+
+	/** @var string */
+	private $i18nMessage;
+
+	/** @var array<int,string> */
+	private $i18nParams;
+
+	public function __construct( string $i18nMessage, array $i18nParams ) {
+		parent::__construct();
+		$this->i18nMessage = $i18nMessage;
+		$this->i18nParams = $i18nParams;
+	}
+
+	public function getI18nMessage(): string {
+		return 'exception-' . $this->i18nMessage;
+	}
+
+	public function getI18nParams(): array {
+		return $this->i18nParams;
+	}
+
+}
```

Comment:
Should normally go in a directory called Exception. I know there's only one right now, but we might add more (I was thinking we need a wrapper for `ProcessFailedException` to keep the error message title consistent so the emails are grouped)

Reply:
Done.